client/lib/state-persistence/enhancer.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import { noop, throttle } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getState, setState } from './worker';
+
+const DEFAULT_OPTIONS = {
+	types: {
+		SERIALIZE: 'SERIALIZE',
+		DESERIALIZE: 'DESERIALIZE',
+		STATE_REPLACE: 'STATE_REPLACE'
+	},
+	throttle: 500,
+	maxAge: 604800000
+};
+
+const DESERIALIZE_INIT = '@@DESERIALIZE_INIT';
```

Comment:
Maybe we should namespace it to `calypso/@@DESERIALIZE_INIT`to make it clear that it's our internal event that should not be handled.

https://github.com/reactjs/redux/issues/186